### PR TITLE
SONARGO-52 Create a new repository for common Gradle scripts

### PR DESCRIPTION
**Pull request by petertrr**
[SONARGO-52](https://sonarsource.atlassian.net/browse/SONARGO-52)

[SONARGO-52]: https://sonarsource.atlassian.net/browse/SONARGO-52?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -11,7 +10,6 @@ plugins {
 
 allprojects {
     apply plugin: 'java'
```
Comment (mstachniuk) [inline build.gradle:11]: I guess this plugin should be also removed

File: build.gradle [hunk 2]
```diff
@@ -11,7 +10,6 @@ plugins {
...
-    apply plugin: 'jacoco'
     apply plugin: 'com.jfrog.artifactory'
     apply plugin: 'maven-publish'
     apply plugin: 'signing'
```
Comment (mstachniuk) [inline build.gradle:14]: A few lines below the `gradle.projectsEvaluated {` may be removed (or added to cloud-native.java-conventions)
It adds the following lines output:
```
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
```

**Review by mstachniuk — APPROVED** [review]
LGTM and pleas qa_plugin test before

File: build.gradle (hunk 1 above)
Comment (petertrr) [reply build.gradle:11]: Right now removal causes some additional problems (spotless is applied to all projects and needs `java` there as well), so we should clean it up after other common scripts are added

**Pull request merged**